data/Officer.json

```
@@ -0,0 +1,80 @@
+[
+    {
+        "name": "Joselyn Narvaez",
+        "position": "President",
+        "image": "/imgs/officers/president.JPG",
```

I was having issues loading the images on my computer due to the case sensitivity but it might just be my browser (Brave) or OS (Ubuntu) - this was running fine on my Windows partition but I guess case sensitivity does matter on certain platforms/browsers
```suggestion
        "image": "/imgs/officers/president.jpg",
```